src/main/java/easy/gc_coffee_api/entity/File.java

```diff
@@ -19,6 +19,11 @@ public class File extends BaseDateEntity {
     private String url;
```

Comment:
```suggestion
    @Column(name = "`key`")
    private String key;
```
컨플릭 수정하실 때 참고하시라고 적어둘게요